terra-jupyter-python/Dockerfile

```diff
@@ -75,8 +74,9 @@ RUN pip3 -V \
  && pip3 install tqdm==4.19.4 \
  && pip3 install werkzeug==0.12.2 \
  && pip3 install certifi==2017.4.17 \
- && pip3 install intel-openmp==2018.0.0 \
- && pip3 install mkl==2018.0.3 \
+# TODO having trouble installing intel-openmp and mkl on a Mac. Commenting for now.
```

Comment:
@akarukappadath do you recall any problems with these libraries (`intel-openmp` and `mkl`)? I think they're needed for GATK.

Reply:
No, I don't remember running into any issues with these packages, but I also wasn't running the image locally then. Is it a dependency issue you're seeing?

Reply:
It actually started working on its own so I put it back :)